Source/Infrastructure/AspNet/Startup.cs

```diff
@@ -46,8 +46,8 @@ public void ConfigureServices(IServiceCollection services)
                 .AddMvc();;
             services.Configure<ConnectionStringsOptions>(_configuration);
 
-            _bootResult = services.AddDolittle(_loggerFactory);
-
+            //_bootResult = services.AddDolittle(_loggerFactory); // Sometimes it seems like this goes in an infinite loop with the logger factory for some reason
+            _bootResult = services.AddDolittle(); // this works
```

Comment:
For some reason, sometimes when I try to run a debug session of UserManagement or Admin, this goes in an ininite loop seemingly because of the _loogerFactory. When I remove the argument from the call it works without problem.
@einari